README.md

```diff
@@ -102,27 +102,35 @@ const displayPinMarker = (coords: LngLatLike, offset: Point): Marker => {}
 
 - [TeritorioCluster](#teritoriocluster)
   - [Parameters](#parameters)
+  - [Options](#options)
   - [addEventListener](#addeventlistener)
 
 ### TeritorioCluster
 
 Create a new Maplibre GL JS plugin for feature (cluster / individual marker) rendering
 
 #### Parameters
+| Name      | Type                                                               | Description                                          | Required | Default value |
+|-----------|--------------------------------------------------------------------|------------------------------------------------------|----------|---------------|
+| map     | [`Map`](https://maplibre.org/maplibre-gl-js/docs/API/classes/Map/) | The Map object represents the map on your page       | ✅        | ❌             |
+| source  | `string`                                                           | The ID of the vector tile or GeoJSON source to query | ✅        | ❌             |
```

Comment:
If can be a GeoJSON, is not a String, but an object ?

Reply:
It cannot be a GeoJSON, we are talking about an ID referencing a vector tile OR GeoJSON source. So it's always a string.
Ref: https://maplibre.org/maplibre-gl-js/docs/API/classes/Map/#querysourcefeatures

Reply:
maybe I can rename `source` by `sourceId` in order to clarify the purpose of the field and to stick with Maplibre definition ?

Reply:
```suggestion
| sourceId  | `string`                                                           | The ID of the source. Should be a vector source, preferably of type GeoJSON and not vector tiles | ✅        | ❌             |
```